src/test/regress/expected/scale_factor.out

```diff
@@ -0,0 +1,308 @@
+--    ->  Update on scale_factor_repl a  (cost=10000000000.00..10000000006.98 rows=34 width=28)
+--          ->  Nested Loop  (cost=10000000000.00..10000000006.98 rows=100 width=28)
+--                ->  Seq Scan on scale_factor_repl a  (cost=0.00..1.10 rows=10 width=14)
+--                ->  Materialize  (cost=0.00..2.65 rows=10 width=14)
+--                      ->  Broadcast Motion 2:3  (slice1; segments: 2)  (cost=0.00..2.50 rows=15 width=14)
+--                            ->  Seq Scan on scale_factor_part_distr b  (cost=0.00..2.10 rows=5 width=14)
+--
+select * from get_plan_rows_in_slice($$
+  update scale_factor_repl a set c1 = b.c2 from scale_factor_part_distr b returning *;
+$$, 2);
+       node_name        | plan_rows 
+------------------------+-----------
+ Explicit Gather Motion | 100
+ ModifyTable            | 34
+ Nested Loop            | 100
+ Seq Scan               | 10
+ Materialize            | 10
+(5 rows)
+
+select * from get_motion_snd_recv($$
+  update scale_factor_repl a set c1 = b.c2 from scale_factor_part_distr b returning *;
+$$);
+       node_name        | motion_snd | motion_recv 
+------------------------+------------+-------------
+ Explicit Gather Motion | 3          | 1
+ Broadcast Motion       | 2          | 3
+(2 rows)
+
+-- (Explicit) Redistribute Motion
+-- (fallback to PO)
+--
+--  Delete on scale_factor_part_distr a  (cost=2.43..5.68 rows=5 width=16)
+--    ->  Explicit Redistribute Motion 3:2  (slice2; segments: 3)  (cost=2.43..5.68 rows=4 width=16)
+--          ->  Hash Join  (cost=2.43..5.68 rows=4 width=16)
+--                Hash Cond: (b.c1 = a.c2)
+--                ->  Seq Scan on scale_factor_rand_distr b  (cost=0.00..3.11 rows=4 width=10)
+--                ->  Hash  (cost=2.30..2.30 rows=4 width=14)
+--                      ->  Redistribute Motion 2:3  (slice1; segments: 2)  (cost=0.00..2.30 rows=5 width=14)
+--                            Hash Key: a.c2
+--                            ->  Seq Scan on scale_factor_part_distr a  (cost=0.00..2.10 rows=5 width=14)
+--
+select * from get_plan_rows_in_slice ($$
+  delete from scale_factor_part_distr a using scale_factor_rand_distr b where b.c1=a.c2;
+$$, 2);
+          node_name           | plan_rows 
+------------------------------+-----------
+ Explicit Redistribute Motion | 4
+ Hash Join                    | 4
+ Seq Scan                     | 4
+ Hash                         | 4
+(4 rows)
+
+select * from get_motion_snd_recv ($$
+  delete from scale_factor_part_distr a using scale_factor_rand_distr b where b.c1=a.c2;
+$$);
+          node_name           | motion_snd | motion_recv 
+------------------------------+------------+-------------
+ Explicit Redistribute Motion | 3          | 2
+ Redistribute Motion          | 2          | 3
+(2 rows)
+
+-- Gather Motion: scaleFactor == 1
+-- slice0 in ORCA: scaleFactor == 1
+--
+--  WindowAgg  (cost=0.00..862.00 rows=11 width=12)
```

Comment:
Estimation is not consistent with this example

Reply:
Tests have been fixed